### PR TITLE
Ft-#164798168  Creating And Getting Articles

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

- Creating Articles and Get Articles

#### Description of Task to be completed?
-  A user should be able to view articles and get articles when they have logged in.

#### How should this be manually tested?

- Fetch this branch `git fetch origin ft-create-articles-164798168`
- Checkout to the branch `git checkout  ft-create-articles-164798168`
- Run `npm install`
- Run `npm run start:dev`

#### What are the relevant pivotal tracker stories?
[ #164798168 ](https://www.pivotaltracker.com/story/show/164798168)
Screenshot 2019-05-16 at 15 35 26
Screenshot 2019-05-16 at 15 35 58
Screenshot 2019-05-16 at 15 36 25
Screenshot 2019-05-16 at 15 36 43

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+baseURL=https://ah-backend.herokuapp.com/api
```
Comment (dannylwe) [inline .env]: @felixkiryowa do not push the .env file

File: .eslintrc.js [hunk 2]
```diff
@@ -15,13 +15,17 @@ module.exports = {
 	"plugins": ["jest", "react"],
 	"rules": {
 		"no-param-reassign":"off",
+		"no-unused-vars":"off",
 		"max-len": [1, 80, 2],
+		"no- param-reassign":"off",
```
Comment (dannylwe) [inline .eslintrc.js:23]: @felixkiryowa do not touch the eslint file 👍

File: src/components/Articles/article.scss [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+
+div.ReactTags__tags {
+    position: relative;
+}
+
+/* Styles for the input */
+div.ReactTags__tagInput {
+    width: 200px;
+    border-radius: 2px;
```
Comment (dannylwe) [inline src/components/Articles/article.scss:46]: @felixkiryowa convert px to rem

File: src/components/SignupForm/index.jsx [hunk 4]
```diff
@@ -7,8 +7,10 @@ import Button from '../SignupButton';
 const SignupForm = ({
   username, email, password, confirmPassword, handleSubmit, onChange, error,
 }) => (
-  <div>
-    <div className="signStr spacebottom">Create an account</div>
+  <div style={{ paddingLeft: '50px' }}>
+    <p style={{ fontSize: '20px', paddingLeft: '65px' }}>
```
Comment (dannylwe) [inline src/components/SignupForm/index.jsx:11]: @felixkiryowa convert px to rem

File: src/pages/Articles/articles.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+}
+div.ReactTags__tagInput input.ReactTags__tagInputField,
+div.ReactTags__tagInput input.ReactTags__tagInputField:focus {
+    height: 31px;
+    margin: 0;
+    font-size: 12px;
+    width: 100%;
+    border: 1px solid #eee;
+    padding: 0 4px;
+}
```
Comment (dannylwe) [inline src/pages/Articles/articles.scss:31]: convert px to rem

**Review by dannylwe — CHANGES_REQUESTED** [review]

File: src/helpers/axiosInstance.js [hunk 6]
```diff
@@ -1,4 +1,24 @@
...
+// axiosInstance.interceptors.request.use((config) => {
+//   if (localStorage.getItem('accessToken')
+//   && config.headers.Authorization === 'Bearer null') {
+//     config.headers.Authorization = `Bearer ${localStorage.getItem('accessToken')}`;
+//   }
+//   return config;
+// });
+
+// export default axiosInstance;
+
```
Comment (dorothyas) [inline src/helpers/axiosInstance.js]: please remove this commented out code

File: src/pages/Articles/tests/__snapshots__/createArticlePage.test.jsx.snap [hunk 7]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Articles Page Component Should render without crashing 1`] = `ReactWrapper {}`;
```
Comment (dorothyas) [inline src/pages/Articles/tests/__snapshots__/createArticlePage.test.jsx.snap:3]: it appears the component of this test was not rendered.

File: src/pages/Dashboard/tests/__snapshots__/Dashboard.test.jsx.snap [hunk 8]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Dashboard Component should render without crashing 1`] = `ReactWrapper {}`;
```
Comment (dorothyas) [inline src/pages/Dashboard/tests/__snapshots__/Dashboard.test.jsx.snap:3]: this component as well was not rendered

File: src/pages/singleArticle/tests/__snapshots__/singleArticle.test.jsx.snap [hunk 9]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Single Article page should render without crashing 1`] = `ReactWrapper {}`;
```
Comment (dorothyas) [inline src/pages/singleArticle/tests/__snapshots__/singleArticle.test.jsx.snap:3]: look into this snapshot as well

**Review by dorothyas — APPROVED** [review]

**Review by dannylwe — APPROVED** [review]

**Pull request merged**